pkg/object/object.go

```diff
@@ -39,6 +39,8 @@ type rwObject struct {
 
 	payloadChecksum *refs.Checksum
 
+	payload []byte
```

Comment:
What do you need payload field for? Last time we checked it, there was no good examples, because most of the time payload will be obtained and processed separately from object structure.

Reply:
I finally decided to add a field so that the SDK users could still read the payload from the object. When working with a payload detached from the object body, this field can remain empty without breaking the general logic.